bcda/auth/models.go

```diff
@@ -24,24 +25,28 @@ func InitializeGormModels() *gorm.DB {
 
 type Token struct {
 	gorm.Model
-	UUID   uuid.UUID   `gorm:"primary_key" json:"uuid"` // uuid
-	User   models.User `gorm:"foreignkey:UserID;association_foreignkey:UUID"`
-	UserID uuid.UUID   `json:"user_id"`                                // user_id
-	Value  string      `gorm:"type:varchar(511); unique" json:"value"` // value
-	Active bool        `json:"active"`                                 // active
-	Token jwt.Token    `gorm:"-"`									   // ignore; not for database
+	UUID        uuid.UUID   `gorm:"primary_key" json:"uuid"` 				// uuid
+	User        models.User `gorm:"foreignkey:UserID;association_foreignkey:UUID"`
+	UserID      uuid.UUID   `json:"user_id"`                                // user_id
+	Value       string      `gorm:"type:varchar(511); unique" json:"value"` // value
+	Active      bool        `json:"active"`                                 // active
+	Token       jwt.Token   `gorm:"-"`                                      // ignore; not for database
+	TokenString string      `gorm:"-"`                                      // ignore; not for database
+	// why are we not storing the AcoID on token?
```

Comment:
The ACO ID is part of the token's value, and we have an ACO association on User.

Reply:
See note below to Knoll's comment on the same question.